content/docs/overview/tracers.md

```diff
@@ -1,3 +1,63 @@
 ---
 title: Tracers
 ---
+
+OpenTracing provides an open, vendor-neutral standard for distributed tracing systems instrumentation, providing an interface to the different distributed tracers. This means that instrumentation would remain the same irrespective of the tracer system being used by the developer. In order to instrument the web application using OpenTracing specification, a compatible OpenTracing tracer must be deployed and listening for incoming span requests. A list of the all the OpenTracing tracers is available [here](./../../supported-tracers). The following section discusses Global Tracers and the various tracer properties that must be taken into account while choosing one.
+
+<br>
+
+## Choosing a Tracer
+It is important that a tracer that meets your tracing requirements as well as fits the needs of your application is chosen. This section discusses some of the parameters for choosing a tracer.
+#### OT compatibility
+The tracer clients must be compatible with the OT API, in order to be used with OpenTracing API instrumentation. It is required that all the OT compatible tracers provide the implementation of tracer interface.
+
+#### OT compatible clients
+It is usually better to choose a tracer with better OT support and a larger diversity of OT-compatible clients for various programming languages. Jaeger is highly OT compatible and has clients available in most of the programming languages.
+
+#### Storage support
+The chosen tracer must be backed by high performance storage support. For production application, Elasticsearch or Cassandra are the most suitable options. MySQL, on the other hand, is not recommended due to performance issues. If persistence is not required, in-memory can also be used by the tracer.
+
+#### Sampling Rate
+The sampling rate of the tracer can either be fixed or dynamic. Dynamic sampling rate supports rate limiting and probabilistic sampling strategies whereas fixed sampling only supports probabilistic sampling strategies. Dynamic sampling rates can adapt to changes in the network behaviour. Jaeger supports dynamic sampling.
+
+<br>
+## Global tracers
+The first step in instrumenting your application with OT specification is registering the tracer with GlobalTracer. This step basically indicates the tracer’s endpoint and the service that will send data to the server. The GlobalTracer forwards all the methods to the tracer that has been configured. The code looks as follows (the tracer is Jaeger):
+
+
+
+
+```java						
```

Comment:
For the overview sections, we should avoid code examples. Those can go in the individual language guides.